playwright-e2e/fixtures/create-fixture.ts

```diff
@@ -17,6 +17,7 @@ import { ManageHearings } from "../pages/manage-hearings";
 import { GatekeepingListing } from "../pages/gatekeeping-listing";
 import { CaseDetails } from "../pages/case-details";
 import { RespondentDetails } from "../pages/respondent-details";
+import { UploadDocuments } from "../pages/upload-documents";
```

Comment:
I believe this has already been added with AddApplicationDocuments page and ticket: https://tools.hmcts.net/jira/browse/DFPL-2150